### PR TITLE
[DOC release] Add documentation for `Ember.ViewMixin.attributeBindings`

### PR DESCRIPTION
**Pull request by srvance**
Closely modeled after the docs for `classNameBindings`.

### CODE REVIEW

File: packages/ember-views/lib/mixins/view_support.js [hunk 1]
```diff
@@ -60,6 +60,52 @@ runInDebug(() => {
  @private
 */
 export default Mixin.create({
+  /**
+    A list of properties of the view to apply as attributes. If the property
+    is a string value, the value of that string will be applied as the value
+    for an attribute of the property's name.
+
+    The following example creates a tag like `<div property="high" />`.
```
Comment (rwjblue) [inline packages/ember-views/lib/mixins/view_support.js]: Should be priority right?

Comment (srvance) [inline packages/ember-views/lib/mixins/view_support.js]: Fixed.

**Comment by srvance** [comment]
Apologies for the git messiness. Tried to simplify with an amend commit and ended up creating a bit of a mess.

**Comment by locks** [comment]
@srvance can you squash the commits while you're at it?

**Comment by srvance** [comment]
@locks Done.

**Review by Turbo87 — APPROVED** [review]

**Pull request merged**